src/dune_rules/modules.mli

```diff
@@ -55,6 +55,8 @@ val fold_user_written : t -> f:(Module.t -> 'acc -> 'acc) -> init:'acc -> 'acc
 val map_user_written :
   t -> f:(Module.t -> Module.t Memo.Build.t) -> t Memo.Build.t
 
+val fold_user_visible : t -> f:(Module.t -> 'acc -> 'acc) -> init:'acc -> 'acc
```

Comment:
I agree it's confusing. Visibility here refers to the library developer. Perhaps it should be `fold_user_available` instead of `visibile`? Since `visibility` already refers to public/private for modules.

Reply:
Fixed now.